### PR TITLE
Explicitly print gitignored

### PR DESCRIPTION
**Pull request by evanrittenhouse**
Resolves: python-poetry/poetry#6443

- [ ] Added **tests** for changed code.
- [ ] Updated **documentation** for changed code.

Didn't add tests as it's just a matter of calling the logger. I also fixed a typo, changing `explicitely_<excluded | included>` to `explicitly_<excluded | included>`.

PR for https://github.com/python-poetry/poetry/issues/6443

### CODE REVIEW

**Comment by dimbleby** [comment]
you seem not to have logged the vcs_ignored_files - which are the ones you actually cared about in https://github.com/python-poetry/poetry/issues/6443

perhaps it would be better just to log the final answer from `ignored`?

**Comment by evanrittenhouse** [comment]
Thanks for the comment! Fixed. Curious under what conditions a file will be explicitly_excluded? l see that it's an instance field in `ProjectPackage`, but I'm wondering what that looks like when actually using Poetry.

**Comment by neersighted** [comment]
> Thanks for the comment! Fixed. Curious under what conditions a file will be explicitly_excluded? l see that it's an instance field in `ProjectPackage`, but I'm wondering what that looks like when actually using Poetry.

There is an `exclude =` field in `pyproject.toml` to explicitly exclude files.

**Comment by evanrittenhouse** [comment]
Curious how the Poetry milestone system works.

**Comment by evanrittenhouse** [comment]
Does being added to the milestone mean that this is good to go, or is more work required?

**Comment by dimbleby** [comment]
this currently has merge conflicts, it's certainly not good to go before they are resolved

**Comment by evanrittenhouse** [comment]
Sorry, missed the merge conflicts - just resolved them @dimbleby

File: src/poetry/core/masonry/builders/builder.py [hunk 1]
```diff
@@ -111,25 +111,29 @@ def find_excluded_files(self, fmt: str | None = None) -> set[str]:
...
 
                 included_glob = inc["path"]
                 for included in self._path.glob(str(included_glob)):
-                    explicitely_included.add(
+                    explicitly_included.add(
                         Path(included).relative_to(self._path).as_posix()
                     )
 
-            ignored = (vcs_ignored_files | explicitely_excluded) - explicitely_included
+            result = set()
```
Comment (dimbleby) [inline src/poetry/core/masonry/builders/builder.py:132]: `result` has been removed altogether from the main branch, this should not reintroduce it

**Comment by evanrittenhouse** [comment]
@dimbleby just bumping this. No rush.

**Comment by dimbleby** [comment]
nothing I can do with it, I'm just a humble contributor

**Comment by evanrittenhouse** [comment]
@neersighted bumping this

**Review by neersighted — APPROVED** [review]

**Pull request merged**